packages/cf-deploy-config-inquirer/src/types.ts

```diff
@@ -0,0 +1,48 @@
+import type { YUIQuestion } from '@sap-ux/inquirer-common';
+import type { AutocompleteQuestionOptions } from 'inquirer-autocomplete-prompt';
+
+export enum promptNames {
+    /** The prompt for the destination name. */
+    destinationName = 'destinationName',
+    /** The prompt for adding an managed app router. */
+    addManagedApprouter = 'addManagedApprouter',
+    /** The prompt for overwriting the destination. */
+    overwrite = 'overwrite'
+}
+
+export interface CfDeployConfigPromptOptions {
```

Comment:
Some of these dont seem to be prompt options. i.e. An optional configuration of a prompt. Can you please rename them so they are clearly specifying an option for a specific prompt. They are not all used in all prompts and so there will be no proper type checking of the prompt name -> option mappping!

Reply:
Updated [here](https://github.com/SAP/open-ux-tools/pull/2436/files#diff-24301133b22fc485c7fb65d11b3e76c2150400123d73802940fe378e8b378ebdR39) as suggested ...